cmake/modules/AddSwift.cmake

```diff
@@ -739,6 +739,10 @@ function(add_libswift name)
       get_filename_component(swift_exec_bin_dir ${ALS_SWIFT_EXEC} DIRECTORY)
       set(sdk_option ${sdk_option} "-resource-dir" "${swift_exec_bin_dir}/../bootstrapping0/lib/swift")
     endif()
+  elseif(${LIBSWIFT_BUILD_MODE} STREQUAL "CROSSCOMPILE-WITH-HOSTLIBS")
+    set(sdk_option "-sdk" "${SWIFT_SDK_${SWIFT_HOST_VARIANT_SDK}_ARCH_${SWIFT_HOST_VARIANT_ARCH}_PATH}")
+    get_filename_component(swift_exec_bin_dir ${ALS_SWIFT_EXEC} DIRECTORY)
+    set(sdk_option ${sdk_option} "-resource-dir" "${swift_exec_bin_dir}/../lib/swift")
```

Comment:
Any reason you don't use this final resource directory, instead of the first bootstrap directory above on Darwin?

Reply:
The final resource dir will contain the libswiftCore. It's important that with this build, the resource dir does contain any swift libraries. They should be picked up from the SDK.

Reply:
By libraries "picked up from the SDK," I assume you mean at runtime? But since libswift is a static library and this resource directory is only used at build time, I'd think the SDK libraries like `swiftCore` are irrelevant.

In any case, looks like you're overhauling the libswift build in #40308 now: I'll wait till that settles out before updating this to adapt to that pull.

Reply:
It's also about the swiftmodule.